sfall/LoadGameHook.cpp

```diff
@@ -276,6 +277,45 @@ static void __declspec(naked) LoadGame() {
 		retn;
 	}
 }
+
+void ReadExtraGameMsgFiles()
+{
+	int read;
+	std::string names;
+
+	names.resize(256);
+
+	while ((read = GetPrivateProfileStringA("Misc", "ExtraGameMsgFileList", "",
+		(LPSTR)names.data(), names.size(), ".\\ddraw.ini")) == names.size() - 1)
+		names.resize(names.size() + 256);
+
+	if (names.empty())
+		return;
+
+	names.resize(names.find_first_of('\0'));
+	names.append(",");
+
+	int begin = 0;
+	int end;
+	int length;
+
+	while ((end = names.find_first_of(',', begin)) != std::string::npos)
+	{
+		length = end - begin;
+
+		if (length > 0)
+		{
+			std::string path = "game\\" + names.substr(begin, length) + ".msg";
```

Comment:
Including string header does solve the error messages on LoadGameHook.cpp, but for some reason there are still lots of errors on FalloutEngine.h. It's weird because it doesn't get changed and I can compile the latest develop commit without any issues.

It would be ideal to if we can keep sfall compatible with VS2008 Express, which uses older C++ standard (C++03 AFAIK). Phobos uses VS2010, and I use both VS2010/2008 Express for compiling/releasing sfall packages.